en/technical-reference/docs-changelog.md

```diff
@@ -3,6 +3,12 @@
 
 <!-- #### Unreleased 
 -->
+#### 2022/07/08
+- add content: email notification snap validation error
+- add content: customer able to retry payment if status order is expired
+- add content: one order have 2 payments
+- remove content: remove mechanic app due to restock doesn't work
```

Comment:
Read Readme. This changelog is aimed for public audience, it should not just a copy paste from commit message. You don't even mention that the changes is specific to Snap guide.